src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -354,15 +355,64 @@ export const ArtworkSidebarCommercialButtons: React.FC<ArtworkSidebarCommercialB
     artwork.isAcquireable || activePartnerOffer
   )
 
+  const SaleMessageOrOfferDisplay: FC = () => {
+    if (activePartnerOffer) {
+      return (
+        <>
+          <Spacer y={2} />
+          <Flex>
+            <Text
+              variant="xs"
+              color="blue100"
+              backgroundColor="blue10"
+              px={0.5}
+            >
+              Limited-Time Offer
+            </Text>
+          </Flex>
+
+          <Spacer y={0.5} />
+
+          <Flex
+            flexDirection={["column", "row"]}
+            width="100%"
+            justifyContent="start"
+          >
+            <SaleMessage
+              saleMessage={activePartnerOffer.priceWithDiscount?.display}
+            />
+            <Spacer x={1} />
+            <Text variant="md" color="black60" style={{ whiteSpace: "nowrap" }}>
+              {/* TODO: Move this logic of display to Artwrok MP model */}
+              (List price: {artwork.price || "Not publicly listed"})
+            </Text>
```

Comment:
:+1: Good to clarify this, I wonder if it is worth raising at web or platform practice